modules/options/src/main/scala/scala/build/options/JmhOptions.scala

```diff
@@ -1,9 +1,28 @@
 package scala.build.options
 
+import scala.build.internal.Constants
+
+/** @param jmhVersion
+  *   the version of JMH to be used in the build
+  * @param enableJmh
+  *   toggle for enabling JMH dependency handling in the build (overrides [[runJmh]] when disabled)
+  * @param runJmh
+  *   toggle for whether JMH should actually be runnable from this build
+  */
 final case class JmhOptions(
-  addJmhDependencies: Option[String] = None,
+  jmhVersion: Option[String] = None,
+  enableJmh: Option[Boolean] = None,
   runJmh: Option[Boolean] = None
```

Comment:
this is rather ugly, but we do need something temporary to track if JMH is enabled, while the `runJmh` property is toggled on and off in builds.
We should be able to simplify this when running JMH will be extracted out of the `run` sub-command (or out of the `main` scope).